components/markdown/markdown.jsx

```diff
@@ -13,7 +13,7 @@ export default class Markdown extends React.PureComponent {
         /*
          * An object mapping channel names to channels for the current team
          */
-        channelNamesMap: PropTypes.object.isRequired,
```

Comment:
this `isRequired` had to be removed as, in `Markdown` component render in `formatting.tsx` file, was expected all the required props to be passed.